tests/foreman/ui/test_hostunification.py

```diff
@@ -254,8 +259,14 @@ def test_positive_rename_content_host(self):
                     new_name=new_name,
                 )
                 self.assertIsNone(self.contenthost.search(vm.hostname))
+                if bz_bug_is_open(1495271):
+                    session.dashboard.navigate_to_entity()
                 self.assertIsNotNone(self.contenthost.search(new_name))
+                if bz_bug_is_open(1495271):
```

Comment:
I think that one skip statement is redundant as we have redirection there already:
`self.contenthosts` -> `self.hosts`

Reply:
Redundant skip removed.